plugins/plugin-core-support/src/lib/cmds/screenshot.ts

```diff
@@ -377,37 +347,49 @@ export default async (commandTree: CommandRegistrar) => {
-                .resize({ width: widthPx, height: heightPx })
-                .toDataURL()}) no-repeat center bottom/contain`
-              snapImg.style.width = widthVw
-              snapImg.style.height = heightVw
+
+              snapImg.setAttribute('src', img.toDataURL())
               snapImg.classList.add('screenshot-image')
 
+              snapContent.classList.add('screenshot-content')
               message.classList.add('screenshot-success-message')
               message.innerText = 'Screenshot copied to clipboard'
+              messageContent.classList.add('screenshot-message')
+              messageContent.appendChild(message)
+              const closeMessage = document.createElement('div')
+              closeMessage.classList.add('screenshot-closing-message')
+              closeMessage.setAttribute('id', 'close-msg')
+              closeMessage.innerHTML = 'Closing in <span id="countdown">10</span>'
+              messageContent.appendChild(closeMessage)
+              snapContent.append(messageContent)
+              snapContent.appendChild(saveButton)
+
+              const closeButton = document.createElement('div')
+
+              close.classList.add('screenshot-close')
+              close.appendChild(closeButton)
 
-              check.classList.add('screenshot-check-icon')
-              const checkIcon = document.createElement('div')
-              checkIcon.innerHTML =
-                '<svg focusable="false" preserveAspectRatio="xMidYMid meet" style="will-change: transform;" xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 32 32" aria-hidden="true"><path d="M16 2C8.3 2 2 8.3 2 16s6.3 14 14 14 14-6.3 14-14S23.7 2 16 2zm-2 19.5l-5-5 1.6-1.5 3.4 3.4 7.4-7.4 1.6 1.6-9 8.9z"></path><path d="M14 21.5l-5-5 1.6-1.5 3.4 3.4 7.4-7.4 1.6 1.6-9 8.9z" data-icon-path="inner-path" opacity="0"></path></svg>'
-              check.appendChild(checkIcon)
+              closeButton.innerHTML =
+                '<div style="display: flex;justify-content:flex-end;align-items:baseline"><svg focusable="false" preserveAspectRatio="xMidYMid meet" style="will-change: transform; cursor:pointer" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 32 32" aria-hidden="true"><path d="M24 9.4L22.6 8 16 14.6 9.4 8 8 9.4l6.6 6.6L8 22.6 9.4 24l6.6-6.6 6.6 6.6 1.4-1.4-6.6-6.6L24 9.4z" stroke="white" fill="white"></path></svg></div>'
+              closeButton.classList.add('screenshot-close-botton')
+
+              // go away after 10 seconds
+              let timeleft = 10
+              downloadTimer = setInterval(() => {
+                if (timeleft <= 0) {
+                  clearInterval(downloadTimer)
+                  finish()
+                }
+                document.getElementById('countdown').innerText = String(timeleft)
+                timeleft -= 1
+              }, 1000)
 
-              // temporarily disable the repl
+              // temporarily disable the repls
```

Comment:
if we want the screenshot popup to be non-modal, shouldn't we remove this disable bit entirely?